release/release_data_tests.yaml

```diff
@@ -421,6 +421,20 @@
       python dataset/gpu_batch_inference.py
       --data-directory 300G-image-data-synthetic-raw-parquet --data-format parquet --chaos-test
 
+- name: batch_inference_chaos_no_scale_back
+  stable: False
+  working_dir: nightly_tests
+
+  cluster:
+    cluster_compute: dataset/autoscaling_gpu_compute.yaml
+
+  run:
+    timeout: 1800
+    prepare: python setup_cluster_compute_config_updater.py --updates worker_nodes.0.max_nodes:5:240
```

Comment:
On second thought, I think you were right -- we might need to scale this up to get useful results.

Do you want to merge this PR, and then I scale up the test as a follow up? Might take me a few days to set it up

Reply:
Yea, I feel this is too small scale.